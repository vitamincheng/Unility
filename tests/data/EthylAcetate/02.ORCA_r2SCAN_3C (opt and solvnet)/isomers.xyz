14
Coordinates from ORCA-job template
  C          -2.14397885950000     -0.60039017560000      0.33140251680000
  C          -0.88003104740000      0.12078782110000     -0.03683251220000
  H          -2.08372214680000     -1.64022467460000      0.02741075220000
  H          -2.99299317650000     -0.10546781600000     -0.13174306270000
  H          -2.26785082320000     -0.54693175130000      1.41169939060000
  O          -0.76679628560000      1.32430611160000     -0.11300547260000
  O           0.11527085070000     -0.73189195790000     -0.27177618400000
  C           1.40908778910000     -0.19949946950000     -0.59538436520000
  C           2.22427365300000      0.06854876810000      0.66098239770000
  H           1.27500251840000      0.72192013740000     -1.17072549690000
  H           1.87812487660000     -0.96582499210000     -1.21515824870000
  H           1.74546056940000      0.84009263470000      1.26014388900000
  H           3.21721524940000      0.40705715910000      0.38027553130000
  H           2.30955799770000     -0.83880374460000      1.25407140590000
14
Coordinates from ORCA-job template
  C          -2.14144271490000     -0.73890698940000      0.23794010580000
  C          -0.97915960690000      0.16322495660000     -0.05862823050000
  H          -2.92268030990000     -0.17267651990000      0.73749324460000
  H          -1.81753868280000     -1.57764820380000      0.84526621060000
  H          -2.54107613180000     -1.10732939950000     -0.70530157120000
  O          -1.06700638580000      1.30362697920000     -0.45957458590000
  O           0.18328979360000     -0.44374381260000      0.16035514900000
  C           1.38383411880000      0.29836020360000     -0.10803003460000
  C           2.57307290420000     -0.57988491330000      0.23093365440000
  H           1.36970665990000      1.21209216830000      0.49603181460000
  H           1.38360319100000      0.58894977640000     -1.16423338890000
  H           2.54404626830000     -0.86060535980000      1.28028999800000
  H           3.49067019080000     -0.03451410140000      0.03205545140000
  H           2.55531779460000     -1.48292974500000     -0.37321834970000
14
Coordinates from ORCA-job template
  C          -2.13276165620000     -0.69310801820000      0.33524773340000
  C          -0.96496307570000      0.16136133990000     -0.06530148690000
  H          -3.03184846800000     -0.08650148580000      0.37722279370000
  H          -1.93458834140000     -1.14808889590000      1.30213914160000
  H          -2.25624432850000     -1.48988556870000     -0.39464070470000
  O          -1.03475257720000      1.32787959490000     -0.38464662100000
  O           0.17178771550000     -0.52845896250000     -0.02825254150000
  C           1.39815131320000      0.13566158890000     -0.37340037830000
  C           2.51375136220000     -0.40234804300000      0.50521310130000
  H           1.26080599590000      1.21368061520000     -0.24030990820000
  H           1.59758721600000     -0.06724723440000     -1.43084954050000
  H           2.60505280570000     -1.47741360110000      0.37515411240000
  H           2.30584727430000     -0.19244665100000      1.55136071140000
  H           3.44970211200000      0.07263194340000      0.22661455440000
