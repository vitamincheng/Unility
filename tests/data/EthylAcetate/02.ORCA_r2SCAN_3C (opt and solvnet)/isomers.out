
       ╔════════════════════════════════════════════╗
       ║            ___ ___ ___ ___ _____           ║
       ║           / __| _ \ __/ __|_   _|          ║
       ║          | (__|   / _|\__ \ | |            ║
       ║           \___|_|_\___|___/ |_|            ║
       ║                                            ║
       ║  Conformer-Rotamer Ensemble Sampling Tool  ║
       ║          based on the xTB methods          ║
       ║                                            ║
       ╚════════════════════════════════════════════╝
       Version 3.0.2, Sun, 25 August 20:02:44, 08/25/2024
       commit (af7eb99) compiled by 'usr@fv-az732-492'

   Cite work conducted with this code as

   • P.Pracht, F.Bohle, S.Grimme, PCCP, 2020, 22, 7169-7192.
   • S.Grimme, JCTC, 2019, 15, 2847-2862.
   • P.Pracht, S.Grimme, C.Bannwarth, F.Bohle, S.Ehlert,
     G.Feldmann, J.Gorges, M.Müller, T.Neudecker, C.Plett,
     S.Spicher, P.Steinbach, P.Wesołowski, F.Zeller,
     J. Chem. Phys., 2024, 160, 114110.

   for works involving QCG cite

   • S.Spicher, C.Plett, P.Pracht, A.Hansen, S.Grimme,
     JCTC, 2022, 18 (5), 3174-3189.
   • C.Plett, S. Grimme,
     Angew. Chem. Int. Ed. 2023, 62, e202214477.

   for works involving MECP screening cite

   • P.Pracht, C.Bannwarth, JCTC, 2022, 18 (10), 6370-6385.

   Original code
     P.Pracht, S.Grimme, Universität Bonn, MCTC
   with help from (alphabetical order):
     C.Bannwarth, F.Bohle, S.Ehlert, G.Feldmann, J.Gorges,
     S.Grimme, C.Plett, P.Pracht, S.Spicher, P.Steinbach,
     P.Wesolowski, F.Zeller

   Online documentation is available at
   https://crest-lab.github.io/crest-docs/

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
   GNU Lesser General Public License (LGPL) for more details.

 Command line input:
 $ crest isomers.xyz --cregen isomers.xyz --rthr 0.175 -- bthr 0.03--ethr 0.15 --ewin 4

  --cregen : CREGEN standalone usage. Sorting file <isomers.xyz>
  --rthr 0.175
  --ewin 4

> Setting up backup calculator ... done.
 ----------------
 Calculation info
 ----------------
> User-defined calculation level:
 : xTB calculation via tblite lib
 : GFN2-xTB level
 :   Molecular charge    : 0
 :   Fermi temperature   : 300.00000
 :   Accuracy            : 1.00000
 :   max SCC cycles      : 500

 Using only the cregen sorting routine.
 input  file name : isomers.xyz
 output file name : isomers.xyz.sorted
 number of atoms                :   14
 number of points on xyz files  :   3
 RMSD threshold                 :   0.1750
 Bconst threshold               :   0.0100
 population threshold           :   0.0500
 # fragment in coord            :     1
 # bonds in reference structure :    13
 number of reliable points      :     3
 sorting energy window (EWIN)   :    4.0000 / kcal*mol⁻¹
 reference state Etot           :  -307.65065846237599     
CREGEN> running RMSDs ... done.
 number of doubles removed by rot/RMSD         :           0
 total number unique points considered further :           3
       Erel/kcal        Etot weight/tot  conformer     set   degen     origin
       1   0.000  -307.65066    0.38895    0.76691       1       2     
       2   0.017  -307.65063    0.37796                                
       3   0.304  -307.65017    0.23309    0.23309       2       1     
T /K                                  :   298.15
E lowest                              :  -307.65066
ensemble average energy (kcal)        :    0.077
ensemble entropy (J/mol K, cal/mol K) :    8.934    2.135
ensemble free energy (kcal/mol)       :   -0.637
population of lowest in %             :   76.691
 number of unique conformers for further calc            2
 list of relative energies saved as "crest.energies"

 -----------------
 Wall Time Summary
 -----------------
 CREST runtime (total)               0 d,  0 h,  0 min,  0.019 sec
 ------------------------------------------------------------------
 CREGEN                     ...        0 min,  0.018 sec ( 95.005%)
 I/O and setup              ...        0 min,  0.001 sec (  4.995%)
 ------------------------------------------------------------------
 * wall-time:     0 d,  0 h,  0 min,  0.019 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.251 sec
 * ratio c/w:    13.478 speedup
 ------------------------------------------------------------------
 CREST terminated normally.
